bench_test.go

```
@@ -29,7 +29,7 @@ func BenchmarkQueue_Operations(b *testing.B) {
 		worker := NewWorker(task)
 		// Bind the worker to a standard queue
 		q := worker.BindQueue()
-		defer q.WaitAndClose()
+		defer worker.WaitAndStop()
```

golangci-lint (1.64.8)

Error return value of `worker.WaitAndStop` is not checked

(errcheck)